wiki/Guides/OpenGL_Support_Issues/de.md

```diff
@@ -1,30 +1,30 @@
 # Probleme mit GL Unterstützung
 
-Die nächste große Veröffentlichung von osu! (kommt in August-September 2015) entfernt die DirectX Unterstützung, um unseren Framework zu vereinfachen. Wenn Sie diesen Artikel sehen, bedeutet es, dass Sie, wenn der neue Build veröffentlich wurde, nicht in der Lage sind osu! auf Ihren derzeitigen System zu spielen. Bitte lesen Sie sich die folgenden Informationen gründlich durch und versuchen Sie sie aus.
+Die nächste große Veröffentlichung von osu! (kommt in August-September 2015) entfernt die DirectX Unterstützung, um unser Framework zu vereinfachen. Wenn du diesen Artikel siehst, bedeutet es, dass du, wenn der neue Build veröffentlich wurde, nicht in der Lage bist osu! auf deinem derzeitigen System zu spielen. Bitte lese dir die folgenden Informationen gründlich durch und versuche sie aus.
```

Comment:
```suggestion
Die nächste große Veröffentlichung von osu! (kommt in August-September 2015) entfernt die DirectX Unterstützung, um unser Framework zu vereinfachen. Wenn du diesen Artikel siehst, bedeutet es, dass du, wenn der neue Build veröffentlich wurde, nicht in der Lage bist osu! auf deinem derzeitigen System zu spielen. Bitte lese dir die folgenden Informationen gründlich durch und probiere sie aus.
```

I know you took the original wording verbatim, but I'm 99% sure `ausversuchen` is not  a real word. Might as well fix this while we're here.